tests/collapse-guess.rs

```diff
@@ -69,6 +70,7 @@ fn collapse_guess_unknown_format_should_log_error() {
     test_collapse_guess_logs(
         "./tests/data/collapse-guess/unknown-format.txt",
         |captured_logs| {
+            println!("{:?}", captured_logs);
```

Comment:
Was this included intentionally?

Reply:
No. Forgot to take it out. Will remove it.